examples/elasticsearch/src/test/kotlin/cnt/elasticsearch/BookServiceTest.kt

```diff
@@ -0,0 +1,150 @@
+package cnt.elasticsearch
+
+import cnt.elasticsearch.ElasticSearchConsts.ANALYSE_API
+import cnt.elasticsearch.ElasticSearchConsts.INDEX_API
+import cnt.elasticsearch.ElasticSearchConsts.INDEX_NAME
+import io.quarkus.test.junit.QuarkusTest
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import org.apache.http.util.EntityUtils
+import org.assertj.core.api.Assertions.assertThat
+import org.elasticsearch.client.Request
+import org.elasticsearch.client.Response
+import org.elasticsearch.client.RestClient
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
+import javax.inject.Inject
+
+@QuarkusTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+class BookServiceTest {
+
+    @Inject
+    private lateinit var bookService: BookService
+
+    @Inject
+    private lateinit var elasticSearchClient: RestClient
+
+    private companion object {
+        val book = Book("Title", "Isbn")
+    }
```

Comment:
Not wrong or bad, but you don't absolutely need a companion object here. You could just define the test data as part of the test class. The only downside is that it gets reset with each test, but "less code" > "nanoseconds faster test initialization".
But this as well is just personal preference on my part.

Reply:
Good point!